### PR TITLE
Create Blog “hpe-sustainability-insight-center-api-at-a-glance”

### PR DESCRIPTION
**Pull request by Didier-Lalli**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/hpe-sustainability-insight-center-api-at-a-glance.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+   line-height: 33px;
+   max-width: none;
+}
+</style>
+
+HPE GreenLake provides a new service called HPE Sustainability Insight Center that can assist you in obtaining detailed information about carbon emissions, energy consumption and the cost of the infrastructure that is managed by HPE GreenLake. In this blog post, I will explain how to extract data from this service in a programmatic way using cURL and the HPE Sustainability Insight Center API.
+
+## What is HPE Sustainability Insight Center?
+
+HPE Sustainability Insight Center is a service that runs on HPE GreenLake. I can add it into my workspace from the HPE GreenLake catalogue under the Management & Governance category.
```
Comment (choukden) [inline content/blog/hpe-sustainability-insight-center-api-at-a-glance.md:22]: catalogue (GB) vs catalog (USA)
Not sure if you want to change it to catalog.

File: content/blog/hpe-sustainability-insight-center-api-at-a-glance.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+* **Monitoring IT costs** — Use data available from the HPE Sustainability Insight Center API to rationalize the power consumption of IT assets operations for cost efficiency. This will free up budget for innovative and growth-orientated investments.
+* **Integrating into reporting workflows** — Use data retrieved from the HPE Sustainability Insight Center to incorporate into existing analytics, reporting, dashboards, and forecasting workflows to give your organization a robust understanding of its IT operations.
+
+## Where is the HPE Sustainability Insight Center API documented?
+
+The [API specifications](https://developer.greenlake.hpe.com/docs/greenlake/services/sustainability/public/) are found on the [HPE GreenLake developer portal](https://developer.greenlake.hpe.com/) along with other HPE GreenLake APIs. For HPE Sustainability Insight Center, there are 3 API calls available:
```
Comment (choukden) [inline content/blog/hpe-sustainability-insight-center-api-at-a-glance.md:42]: Second hotlink for HPE GreenLake developer portal

Maybe I would connect readers to the /services page https://developer.greenlake.hpe.com/docs/greenlake/services/ 

Also, I think the spelling should be "HPE GreenLake Developer Portal".

**Review by choukden — COMMENTED** [review]
Just a couple of comments for you to consider if you wish.
Denis

File: content/blog/hpe-sustainability-insight-center-api-at-a-glance.md (hunk 1 above)
Comment (Didier-Lalli) [reply content/blog/hpe-sustainability-insight-center-api-at-a-glance.md]: Ok

File: content/blog/hpe-sustainability-insight-center-api-at-a-glance.md (hunk 2 above)
Comment (Didier-Lalli) [reply content/blog/hpe-sustainability-insight-center-api-at-a-glance.md:42]: Ok merci.

**Pull request merged**